charge.go

```diff
@@ -379,6 +380,36 @@ type ChargePaymentMethodDetailsIdeal struct {
 	VerifiedName string `json:"verified_name"`
 }
 
+// ChargePaymentMethodDetailsInteracPresent represents details about the InteracPresent PaymentMethod.
+type ChargePaymentMethodDetailsInteracPresent struct {
+	Brand          string                                           `json:"brand"`
+	CardholderName string                                           `json:"cardholder_name"`
+	Country        string                                           `json:"country"`
+	EmvAuthData    string                                           `json:"emv_auth_data"`
+	ExpMonth       int64                                            `json:"exp_month"`
+	ExpYear        int64                                            `json:"exp_year"`
+	Fingerprint    string                                           `json:"fingerprint"`
+	Funding        string                                           `json:"funding"`
+	GeneratedCard  string                                           `json:"generated_card"`
+	Last4          string                                           `json:"last4"`
+	Network        string                                           `json:"network"`
+	ReadMethod     string                                           `json:"read_method"`
+	Receipt        *ChargePaymentMethodDetailsInteracPresentReceipt `json:"receipt"`
+}
+
+// ChargePaymentMethodDetailsInteracPresentReceipt represents details about the InteracPresent Receipt.
+type ChargePaymentMethodDetailsInteracPresentReceipt struct {
```

Comment:
I always put "nested" resources above in go so this should be before. Not blocking but FYI